apps/docs/src/app/core/component-docs/checkbox/checkbox-header/checkbox-header.component.html

```diff
@@ -2,7 +2,13 @@
 <description>
     With checkboxes, all options are visible and the user can make one or more selections. This component can be set
     disabled and also displayed in a row. <br />Checkboxes are keyboard navigable via the tab key and selected with the
-    space bar.
+    space bar.<br/>
+    <fd-message-strip [type]="'information'">
+        On new version of chromium engine (Opera, Microsoft Edge, Chrome) checkbox component causes crash.<br>
+        <a fd-link href="https://github.com/SAP/fundamental-styles/issues/1197">There is issue to follow</a>.<br>
```

Comment:
should be: "this is the issue to follow:" <a fd-link href="https://github.com/SAP/fundamental-styles/issues/1197">click here</a>